app/assets/javascripts/admin/addon/components/admin-config-areas/about.gjs

```diff
@@ -6,9 +6,51 @@ import AdminConfigAreasAboutGeneralSettings from "admin/components/admin-config-
 import AdminConfigAreasAboutYourOrganization from "admin/components/admin-config-area-cards/about/your-organization";
 
 export default class AdminConfigAreasAbout extends Component {
-  saveCallback() {
-    // eslint-disable-next-line no-console
-    console.log("save callback");
+  get generalSettings() {
+    const hash = {};
+
+    hash.title = this.#lookupSettingFromData("title");
+    hash.siteDescription = this.#lookupSettingFromData("site_description");
+    hash.extendedSiteDescription = this.#lookupSettingFromData(
+      "extended_site_description"
+    );
+    hash.aboutBannerImage = this.#lookupSettingFromData("about_banner_image");
+
+    return hash;
+  }
```

Comment:
This would be simpler:

```suggestion
  generalSettings = {
    title: this.#lookupSettingFromData("title"),
    siteDescription: this.#lookupSettingFromData("site_description"),
    extendedSiteDescription: this.#lookupSettingFromData(
      "extended_site_description"
    ),
    aboutBannerImage: this.#lookupSettingFromData("about_banner_image"),
  };
```

Reply:
Agree with this too for all 3 hashes.